store/etcd/proto/types.proto

```diff
@@ -0,0 +1,54 @@
+syntax = "proto3";
```

Comment:
We could add a `proto.go` file that would contain Go Generate commands like the following:

```go
package proto

//go:generate protoc --go_out=. types.proto
```

And add a Makefile command `make gen` that would simply do

```
gen:
  go generate ./...
```